### ISSUE TITLE
Feature request: analysis of move can take a long time; provide an abort option

### ISSUE THREAD

**Issue by ACinnamonUserInUK** [lukasmonk/lucaschess#41]
So far as I can see - and I use Lucas Chess on Windows and on Linux under Wine - there's is no way to abort the process that is begun when one clicks 'help to move'. Since that process can take a long time - depending on one's settings and the power of one's computer - i suggest that some sort of option to abort would be a good idea.

**Comment by lukasmonk** [comment]
The time used is the same as the tutor, or to an analysis of a move.
To study, thanks.

**Comment by ACinnamonUserInUK** [comment]
Ah. Still, an abort option would be useful, no? Especially if one has set the time in question to twenty minutes or something like that!

**Comment by lukasmonk** [comment]
Yes, I have to add a Cancel button in all analysis (help to move is internally an analysis).
